components/tidb_query_expr/src/impl_json.rs

```diff
@@ -204,6 +204,35 @@ fn quote(bytes: BytesRef) -> Result<Option<Bytes>> {
     Ok(Some(result))
 }
 
+#[rpn_fn(nullable, raw_varg, min_args = 1)]
+#[inline]
+fn json_valid(args: &[ScalarValueRef]) -> Result<Option<Int>> {
+    assert!(!args.is_empty() && args.len() == 1);
```

Comment:
why not set `min_args`= 1, `max_args` = 1 by `rpn_fn`

Reply:
fixed